src/theory/arith/linear/arith_static_learner.cpp

```diff
@@ -338,9 +346,201 @@ void ArithStaticLearner::addLearnedLemma(TNode n,
 
 std::shared_ptr<ProofNode> ArithStaticLearner::getProofFor(Node fact)
```

Comment:
This method works with a given assumption on the structure of `fact`. I think it'd be good to document this in the header file.